### DISCUSSION TITLE
How to downgrade brew back to 3.0.11?

### DISCUSSION THREAD

**Discussion by jackric** [Homebrew/discussions#1269] [Linux]
I'm having issues using the latest release `3.1.1`, which is trying to get packages from `ghcr.io`.
I have an infrastructure issue blocking `gchr.io` at the moment; as a short term workaround how can I roll back my brew client to 3.0.11?

Thanks

**Comment by SMillerDev** [comment]
With bintray starting to do brownouts, I would really recommend against that because it can essentially stop working at any point in time.

**Comment by carlocab** [comment]
It's probably simpler to set `HOMEBREW_BOTTLE_DOMAIN=https://homebrew.bintray.com/`. Note that we've stopped uploading bottles to Bintray, so you may find yourself unable to install some recently updated formulae. You'll face the same issue from rolling `brew` back to `3.0.11`.
